### DISCUSSION TITLE
Creating payment session with Medusa.JS Client: Could not find a payment provider with id: stripe'

### DISCUSSION THREAD

**Discussion by dianaow** [medusajs/medusa#6591] [Q&A]
I encounter a 404 responses status error: 'Could not find a payment provider with id: stripe' during this request 
"POST /store/carts/cart_01XXX/payment-sessions "

cartid is not null. I have followed the steps [here](https://docs.medusajs.com/plugins/payment/stripe), but will be useful to have a full workflow non-react example with the sequence of API calls to successfully trigger the stripe payment form.

```
   let cart = await medusa.carts.createPaymentSessions(cartid)
      .then(({ cart }) => cart)
      .catch((e) => console.log(e))
```

**Comment by olivermrbl** [comment]
1. Create cart: `POST /store/carts`
2. Select region: `POST /store/carts/:id -> { region_id: "reg_1234" }`
Ensure the selected region has Stripe installed as a payment provider. See more [here](https://docs.medusajs.com/user-guide/regions/providers).
3. Add items to cart: `POST /store/carts/:id/line-items -> { variant_id: "var_1234", quantity: 2 }`
4. *Go to checkout
5. Add personal details: `POST /store/carts/:id -> { shipping_address: { ... }, email: { ... } }`
6. Add shipping: `POST /store/carts/:id/shipping-methods -> { option_id: "so_1234" }`
7. Create payment sessions: `POST /store/carts/:id/payment-sessions`
8. Select Stripe payment session: `POST /store/carts/:id/payment-session { provider_id: "stripe" }`
This step is only required, if you have more than one provider installed.
10. *Enter payment information in Stripe payment form
11. Complete cart: `POST /store/carts/:id/complete`

These are the steps required to collect a payment with Stripe in a conventional checkout flow. 

Judging from your error message, I would say your Stripe plugin is not installed correctly. 

Can you provide a reproducible setup?

**Comment by dianaow** [comment]
I see the 'stripe' option in the medusa admin.

![Screenshot 2024-03-06 at 6 28 40 PM](https://github.com/medusajs/medusa/assets/33983719/e48cf3c1-e680-4307-8b1f-7d48026c63fb)

**Comment by dianaow** [comment]
Actually i resolved it by unchecking "Manual" as a payment provider

**Comment by dianaow** [comment]
This question was closed before I could add that while it resolved the error at creating the payment session, the same error message appears when I select a payment session with stripe as the provider_id.

I have submitted a bug report here: https://github.com/medusajs/medusa/issues/6597

I'm also keen to know the reason why unchecking "Manual" as a payment provider worked, when this is not suggested anywhere in the documentation
